src/main/java/regions/Precinct.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package regions;
+
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import utils.Validator;
+
+/**
+ *
+ * @author Hengqi Zhu
+ */
+@Entity
+@Table(name="PRECINCT")
+public class Precinct implements Serializable {
+    	private static final long serialVersionUID = 1L;

```

Comment:
If you don't change this class during runtime, you don't need it. If you add a field to a class during runtime and serialVersionUID is declared, an exception will be thrown. I prefer to delete it, in case of professor ask...and I cannot explain well.